backend/api/Controllers/DocumentController.cs

```diff
@@ -73,20 +73,33 @@ public IActionResult GetDocumentTypes()
         [SwaggerOperation(Tags = new[] { "documents" })]
         public IActionResult GetDocumentList()
         {
-            var result = _documentService.GetDocumentList();
+            var result = _documentService.GetStorageDocumentList();
             return new JsonResult(result);
         }
 
         /// <summary>
         /// Downloads the file for the correspoding file and document id.
```

Comment:
I realize this was from a previous sprint, but these controllers also were WIP at that time.
nit: correspoding -> corresponding